### ISSUE TITLE
Adds trailing spaces

### ISSUE THREAD

**Issue by FFxSquall** [Prettyhtml/prettyhtml#79]
## 🐛 Bug Report

![60b00042738d755342ed8a6b7afc0bdf](https://user-images.githubusercontent.com/787076/49150777-fe8caa00-f31e-11e8-8d61-d3fe83305a80.png)

## To Reproduce

1. Use vetur and options:
```
    "prettyhtml": {
      "printWidth": 100,
      "singleQuote": false,
      "wrapAttributes": true,
      "sortAttributes": true
    }
```
2. Format document

Paste your markup here:

```html
<template>
  <div>
      <img
        :src="banner.headerImg"
        class="image mb-2 grey lighten-3"
      >
    
      <img
        :src="banner.headerSmallImg"
        class="image mb-2 grey lighten-3"
        v-if="banner.headerSmallImg"
      >
  </div>
</template>
```

## Expected behavior

Paste the results here:

```html
<template>
  <div>
      <img
        :src="banner.headerImg"
        class="image mb-2 grey lighten-3"
      >

      <img
        :src="banner.headerSmallImg"
        class="image mb-2 grey lighten-3"
        v-if="banner.headerSmallImg"
      >
  </div>
</template>

```

## Context

- [ ] Angular
- [ ] Svelte
- [x] Vue
- [ ] Riot
- [ ] HTML5 only

## Your Environment

- Prettyhtml version: 0.8.4
- NodeJs version: 10.13
- Environment name and version: VScode 1.29.1, Vetur 0.14.2
- Operating System and version: Ubuntu 18.04

**Comment by StarpTech** [comment]
Hi @FFxSquall thank you for the bug report. This is a bug.

**Comment by flurmbo** [comment]
Hey @StarpTech mind if I work on this issue?

**Comment by StarpTech** [comment]
Hi @flurmbo thank you very much but I'm already on it and will provide a fix today.

**Comment by flurmbo** [comment]
No problem @StarpTech, glad I asked!

**Comment by StarpTech** [comment]
Fixed in v0.8.6
